docs(Devtools): add manual widget installation (#87)
moovweb-docs/xdn-docs: guides/devtools.md

yarn add --dev @xdn/devtools @xdn/prefetch
```

### Client widget

**Note**: you can skip this step if you are using `@xdn/next` or `@xdn/nuxt`

> Then load the widget in the Browser's javascript

Reviewer: ```suggestion
Add the following to your client JavaScript bundle:
```

```js
import installDevtools from '@xdn/devtools/install'

installDevtools()
```

> Alternatively you can also inject the widget install script directly in your HTML `<head>`

Reviewer: ```suggestion
Alternatively, you can add the following `script` tag to your app's HTML:
```
